cmd/lk/room.go

```diff
@@ -820,11 +835,12 @@ func joinRoom(ctx context.Context, cmd *cli.Command) error {
 		}
 	}
 
-	if cmd.StringSlice("publish") != nil {
+	exitAfterPublish := cmd.Bool("exit-after-publish")
+	if publish := cmd.StringSlice("publish"); publish != nil {
```

Comment:
Just confirming this returns nil and not an empty slice if flag is omitted?

Reply:
I assume so, if it was working before :man_shrugging: